### ISSUE TITLE
Syndicate spy drones

### ISSUE THREAD

**Issue by Kmc2000** [yogstation13/yogstation-classic#341]
So, for the low cost of  3TC, you can buy yourself this little guy (if you think this is unbalanced, comment)
https://gyazo.com/db35f4dd276d8782060671702c39c366

He would function as a scout, with your 3TC you'd also get a neat little drone phone, which you'd use to control it: 
https://gyazo.com/ca04c60245d7d6a35dfe67979615b443

The drone would be able to slip under doors, and have the hide verb that alien larvae have, so you could deploy it under a table and spy on people. He'd have 35 HP, so it would take a few hits to knock it down.

It would also have the ability to tag specific people, and this would go to the drone phone, when you examined it, it would say, for example: Phazon [mime] Last seen: Bridge, and would go away after you tag someone else, to clear all tags you would have to use a special verb on the drone whilst in use.

It'd look like this when tagging:
https://gyazo.com/635740bdb560894cf44ec246f650683b

So you'd be able to hide in a locker, and deploy this little guy to scout ahead for you, you would get the drone and phone TP'd in for your TC and to control the drone, it'd be one click

To stop controlling the drone, a verb in a drone tab next to his hide verb, saying terminate link.

So, why?

Well there aren't an awful lot of USEFUL scouting items, there's the camera bug but that's pretty crappy, having your own drone which you can tag your target with would be pretty cool IMO.

So tl;dr You get a drone which you can control and see through it's vision to tag your target(s)

**Comment by ShadowDeath6** [comment]
Pretty cool, kind of like a less effective camera bug, I guess. 

Nice doublepost lad

**Comment by Super3222** [comment]
rainbow
six
siege
mode
confirmed.

**Comment by Kmc2000** [comment]
HONK

**Comment by FluffySurvivor** [comment]
Kmc is pulling his siege fantasies in spessmen
7 tc is way too much for something that just /scouts/
For 7tc i could already have piled up 5 security corpses :^)

**Comment by Kmc2000** [comment]
@FluffySurvivor 

What about 3 then, making it sleepy pen tier?

**Comment by FluffySurvivor** [comment]
2-3 tc seems good, considering it can be destroyed

**Comment by Kmc2000** [comment]
Cool cool, now, SIEGE STATION 13 WHEN?

**Comment by Super3222** [comment]
I'll start workin' on it.

**Comment by Super3222** [comment]
I'm done with this.

Only thing missing is a resized item and icon state and some action buttons for when you are controlling the damn drone.

**Comment by ghost** [comment]
I'm closing this issue as super's made a PR for it.
